code/modules/research/designs/power_cells.dm

```diff
@@ -65,24 +65,24 @@
 /datum/design/item/powercell/advance
```

Comment:
Your Datums also all conflict, as they're using the same words, as the datums for the APC Cell designs.